src/mscorlib/src/System/IO/StreamReader.cs

```diff
@@ -71,6 +71,9 @@ internal static int DefaultBufferSize
         // This is used only for preamble detection
         private int bytePos;
 
+        [NonSerialized]
+        private StringBuilder builder;
```

Comment:
The existing style should be preferred for changes in the CoreCLR repository -> no underscore here.

From https://github.com/dotnet/coreclr/wiki/Contribution-guidelines

For coding guidelines for this repository, please do give priority to the current style of the project or file you're changing even if it diverges from the guidelines.